pkgs/development/guile-modules/guile-gcrypt/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, guile, libgcrypt
+, autoreconfHook, pkgconfig, texinfo
+}:
+
+let
+  name = "${pname}-${version}";
+  pname = "guile-gcrypt";
+  version = "0.2.1";
+in stdenv.mkDerivation {
+  inherit name;
```

```suggestion
stdenv.mkDerivation {
  pname = "guile-gcrypt";
  version = "0.2.1";
```

Seems strange, since you don't do this anywhere else, and I don't see `name` being used. Also, `mkDerivation` does the `name` stuff for you, if you provide `pname` and `version`...

---

oh, I forget why I had done this before. I think there was a reason, but yes it's quite unnecessary now